### ISSUE TITLE
Cannot Update server using `pushError.Update`

### ISSUE THREAD

**Issue by freshdev1** [Azure/azure-mobile-apps-cordova-client#33]
Hello Everybody,
I had been recently creating an App using Azure Mobile Cordova SDK. My application is build on Ionic 2. The Cordova Plugin is correctly configured and working as expected. Recently I had to implement the `conflict` scenarios. Currently there are only two scenarios that I am catering.

* If user selects the server copy checkbox, the local changes are discarded and server values are stored locally.
* If user selects the local copy checkbox, the local changes are `update` - d - on the server, and retained locally as well.

For the first scenario I run 
```
pushError.cancelAndUpdate(this.serverRecord).then((success) => {
	console.log("In Server copy success:");
	console.log(success);

	this.serverValueSaving = false;
	this.localValueSaving = false;
}, (error) => {
	console.log(error);
});
```
For the second scenario I am doing the following:
```
//this.clientRecord.Version = '';
pushError.update(this.clientRecord).then((success) => {
 
  console.log("In Server update success:");
  console.log(success);

  // pushError.isHandled = true;

  // console.log("Handled value:");
  // console.log(pushError.isHandled);

  this.serverValueSaving = false;
  this.localValueSaving = false;

}, (error) => {
  console.log(error);
});
```
In my understanding this should work and is pretty straight forward. But this does not work. I do get this `In Server update success:` in my console, but the values are not updated on the server, and the conflict is not understood as `handled` by the SDK.

PS:
I added `this.clientRecord.Version = '';` line, because when I tried to run the `PATCH` request through `POSTMAN` with the same payload I was getting `409 (Conflict)`, but if I removed the `Version` value, I got `HTTP OK 200` response. But this is also not working in the above code.

Please help! I am stuck badly! :worried:

**Comment by shrishrirang** [comment]
This is a duplicate of https://github.com/Azure/azure-mobile-apps-js-client/issues/200. Refer https://github.com/Azure/azure-mobile-apps-js-client/issues/200 for details.

**Comment by freshdev1** [comment]
Thanks a lot for the prompt reply. I have left office by now. I did see the sample before, but when I got stuck with this problem the sample didn't come in my mind. :worried: !I believe this line in your sample was what I was missing:                                                                                                                                     ‎https://github.com/shrishrirang/azure-mobile-apps-cordova-client-samples/blob/38da82275c2d3e95bddc18f203d8539b185886f5/offline-sync-error-handling/www/js/tableManager.js#L108But again I believe a new bug should be logged. The SDK should give me some sort of error if this indeed was the problem. Will update you by tomorrow, when I try this at office.Thanks again for the help! :)                                                                                                                                                                                                   Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: Shrirang ShirodkarSent: Thursday, 22 September 2016 12:01 AMTo: Azure/azure-mobile-apps-cordova-clientReply To: Azure/azure-mobile-apps-cordova-clientCc: freshdev1; AuthorSubject: Re: [Azure/azure-mobile-apps-cordova-client] Cannot Update server using `pushError.Update` (#33)Closed #33.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Azure/azure-mobile-apps-cordova-client","title":"Azure/azure-mobile-apps-cordova-client","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Azure/azure-mobile-apps-cordova-client"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #33."}],"action":{"name":"View Issue","url":"https://github.com/Azure/azure-mobile-apps-cordova-client/issues/33#event-797833254"}}}
